### PR TITLE
refactor,chore(egen): refactored code as per template guidelines

### PR DESCRIPTION
**Pull request by Jayakrishna2801**
This PR includes
1.Replace REGION with LOCATION.
2.Removes IS_TESTING condition.
3.Adds colab enterprise logo with link.
4.Makes Heading changes according to template guidelines.
5.Fix unnecessary "``" code highlight usage.
6.Adds Colab authentication and kernal restart steps.
7.removed unnecessary code

**REQUIRED:** Fill out the below checklists or remove if irrelevant
1. If you are opening a PR for `Official Notebooks` under the [notebooks/official](https://github.com/GoogleCloudPlatform/vertex-ai-samples/tree/main/notebooks/official) folder, follow this mandatory checklist:
- [x] Use the [notebook template](https://github.com/GoogleCloudPlatform/vertex-ai-samples/blob/main/notebooks/notebook_template.ipynb) as a starting point.
- [x] Follow the style and grammar rules outlined in the above notebook template.
- [x] Verify the notebook runs successfully in Colab since the automated tests cannot guarantee this even when it passes.
- [x] Passes all the required automated checks. You can locally test for formatting and linting with these [instructions](https://github.com/GoogleCloudPlatform/vertex-ai-samples/blob/main/CONTRIBUTING.md#code-quality-checks).
- [ ] You have consulted with a tech writer to see if tech writer review is necessary. If so, the notebook has been reviewed by a tech writer, and they have approved it.
- [ ] This notebook has been added to the [CODEOWNERS](https://github.com/GoogleCloudPlatform/vertex-ai-samples/blob/main/notebooks/official/CODEOWNERS) file under the `Official Notebooks` section, pointing to the author or the author's team.
- [x] The Jupyter notebook cleans up any artifacts it has created (datasets, ML models, endpoints, etc) so as not to eat up unnecessary resources.

### CODE REVIEW

File: notebooks/official/automl/automl-tabular-classification.ipynb [hunk 1]
```diff
@@ -32,23 +32,26 @@
         "# Vertex AI SDK for Python: AutoML Tabular training and prediction\n",
```
Comment (kittyabs) [inline notebooks/official/automl/automl-tabular-classification.ipynb:32]: - change: "AutoML Tabular "
- to: "AutoML tabular "

**Review by kittyabs — COMMENTED** [review]
small edit

**Review by katiemn — APPROVED** [review]

**Pull request merged**